source/git/source.go

```diff
@@ -539,6 +539,14 @@ func (gs *gitSourceHandler) Snapshot(ctx context.Context, g session.Group) (out
 		if err != nil {
 			return nil, err
 		}
+		_, err = checkoutGit.Run(ctx, "lfs", "install", "--local")
```

Comment:
I think we should check if lfs is supported (git >= 1.8.3.1): https://github.com/git-lfs/git-lfs/wiki/Installation#requirements